Monika After Story/game/script-python.rpy

```diff
@@ -0,0 +1,1205 @@
+#   9 - modules (might be too much) / importing? -- mention importing only, module making is too much
+#   10 - lists
+#   11 - dict
+#   12 - tuples
+#   13 - py2 vs py3
+#   14 - String operations
+#   15 - start talking about renpy
+#
+#
+# Also what about Renpy?
+#
+# Another NOTE: We should try to make the topics of an adequate lenght otherwise
+# we're just going to throw a lot of info that is going to be ignored or forgotten quickly
+# I think splitting something in more than one topic may be a good idea
+#
+## We can assume evhand is already imported
+
+###### tip tree ##############################
+# 0 -> 1
+# 1 -> 3
+# 2 -> 6
+# 3 -> 2
+##############################################
+
+init 4 python in mas_ptod:
+    # to simplify unlocking, lets use a special function to unlock tips
+    import datetime
+    import store.evhand as evhand
+
+    M_PTOD = "monika_ptod_tip{:0>3d}"
+
+    def has_day_past_tip(tip_num):
+        """
+        Checks if the tip with the given number has already been seen and
+        a day has past since it was unlocked.
+        NOTE: by day, we mean date has changd, not 24 hours
+
+        IN:
+            tip_num - number of the tip to check
+
+        RETURNS:
+            true if the tip has been seen and a day has past since it was
+            unlocked, False otherwise
+        """
+        # as a special thing for devs
+        if renpy.game.persistent._mas_dev_enable_ptods:
+            return True
+
+        tip_ev = evhand.event_database.get(
+            M_PTOD.format(tip_num),
+            None
+        )
+
+        if tip_ev is None:
+            return False
+
+        # otherwise, unlocked date is our key
+        if tip_ev.unlock_date is None or tip_ev.shown_count == 0:
+            return False
+
+        # now check the actual day
+        return (
+            datetime.date.today() - tip_ev.unlock_date.date() 
+            >= datetime.timedelta(days=1)
+        )
+
+    def has_day_past_tips(*tip_nums):
+        """
+        Variant of has_day_past_tip that can check multiple numbers
+
+        SEE has_day_past_tip for more info
+
+        RETURNS:
+            true if all the given tip nums have been see nand a day has past
+                since the latest one was unlocked, False otherwise
+        """
+        for tip_num in tip_nums:
+            if not has_day_past_tip(tip_num):
+                return False
+
+        return True
+
+
+# The initial event is getting Monika to talk about python
+# this must be hidden after it has been completed
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_ptod_tip000",
+            category=["python tips"],
+            prompt="Can you teach me about Python?",
+            pool=True
+        )
+    )
+
+label monika_ptod_tip000:
+    m 3eub "You want to learn about Python?"
+    m 3hub "I'm glad!"
```

Comment:
nitpick: "I'm glad!" seems a bit random.  Maybe "I'm so happy you asked me!" to tie it into the next line.